src/components/home/home.component.jsx

```diff
@@ -17,7 +16,7 @@ const Home = () => {
   const classes = useHomeStyles()
 
   return (
-    <Layout>
+    <>
```

Comment:
This wrapper is not needed as we are already returning a `section` below

Reply:
Actually it is needed to have adjacent JSX elements wrapped in an enclosing tag. There are multiple sections below.